tests/test_util.go

```diff
@@ -558,6 +560,54 @@ func createEqualVersionsPredicate(version driver.Version) func(driver.VersionInf
 	}
 }
 
+// clusterSidecarsEqualSpec returns nil if sidecars from spec and cluster match
+func waitUntilClusterSidecarsEqualSpec(t *testing.T, spec api.DeploymentMode, depl api.ArangoDeployment) error {
+
+	c := cl.MustNewInCluster()
+	ns := getNamespace(t)
+
+	var noGood int
+	for start := time.Now(); time.Since(start) < 300*time.Second; {
+
+		// Fetch latest status so we know all member details
+		apiObject, err := c.DatabaseV1alpha().ArangoDeployments(ns).Get(depl.GetName(), metav1.GetOptions{})
+		if err != nil {
+			t.Fatalf("Failed to get deployment: %v", err)
+		}
+
+		// How many pods not matching
+		noGood = 0
+
+		// Check member after another
+		apiObject.ForeachServerGroup(func(group api.ServerGroup, spec api.ServerGroupSpec, status *api.MemberStatusList) error {
+
+			t.Logf("%s group has %d sidecars", group.AsRole(), spec.GetSidecars())
+
+			for _, m := range *status {
+				for _, scar := range spec.GetSidecars() {
+					mcar, found := m.SideCarSpecs[scar.Name]
+					if found {
+						if !reflect.DeepEqual(mcar, scar) {
```

Comment:
sideCarRequireRotation above seems to require a more complex comparison. Is reflect.DeepEqual sufficient here?

Reply:
For the test this is sufficient here, I think.

Reply:
yes is sufficient for the tests